Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -859,18 +859,11 @@ void HTMLInputElement::form_associated_element_attribute_changed(FlyString const
         m_type = parse_type_attribute(value.value_or(String {}));
     } else if (name == HTML::AttributeNames::value) {
```

Comment:
Commit message here as well - just saying "refactor" isn't very clear in what is trying to be achieved (and minor, but on the other refactor commit message, small grammar typo now, looks like it is missing a "to" in the sentence)